src/main/java/uk/co/ramp/Outbreak.java

```diff
@@ -63,171 +70,71 @@ public void setContactRecords(Map<Integer, List<ContactRecord>> contactRecords)
         return records;
     }
 
-
     void generateInitialInfection() {
 
         Set<Integer> infectedIds = initialCaseReader.getCases();
+        List<Event> virusEvents = new ArrayList<>();
 
-        for (Integer id : infectedIds) {
+        InfectionEvent genericEvent = ImmutableInfectionEvent.builder().
+                exposedBy(Case.getInitial()).
+                oldStatus(SUSCEPTIBLE).
+                newStatus(EXPOSED).
+                exposedTime(0).
+                id(-1).
+                time(0).build();
 
-            EvaluateCase evaluateCase = new EvaluateCase(population.get(id), diseaseProperties, distributionSampler);
-            evaluateCase.updateVirusStatus(EXPOSED, 0, Case.getInitial());
+        infectedIds.forEach(id -> virusEvents.add(ImmutableInfectionEvent.builder().from(genericEvent).id(id).build()));
```

Comment:
Personal preference but feel free to ignore:
```suggestion
        infectedIds.forEach(id -> virusEvents.add(ImmutableInfectionEvent.copyOf(genericEvent).withId(id)));
```